ui/src/app.tsx

```diff
@@ -80,6 +146,44 @@ export function App() {
     }
   }
 
+  const generateJsonFileName = (imageName: string) => {
+    const str1 = imageName.split(':').join('.');
```

Comment:
just a question - why did we change this to generate a filename instead of scan.json?

Reply:
this changed because we can have multiple scan files that exist in the docker volume, so we needed a way to uniquely identify them. we could delete old scan files every time we start a new one to guarantee that there aren't any other files, and in that case, we could name it "scan.json" again.